server/portal/settings/settings_secret.example.py

```diff
@@ -71,23 +80,23 @@
 _BROKER_URL_PORT = '5672'
 _BROKER_URL_VHOST = 'dev'
 
-_RESULT_BACKEND_USERNAME = 'dev'
-_RESULT_BACKEND_PWD = 'dev'
-_RESULT_BACKEND_HOST = 'core_portal_redis'
-_RESULT_BACKEND_PORT = '6379'
-_RESULT_BACKEND_DB = '0'
-
 ########################
 # ELASTICSEARCH SETTINGS
 ########################
 
-_ES_HOSTS = 'core_portal_elasticsearch'
+_ES_HOSTS = 'core_portal_elasticsearch:9200'
```

Comment:
__Question?__

Should the comment for an example value (a big object) in Stache be added to these example secrets/settings?

<details><summary>the actual comment content</summary>

```
# _ES_HOSTS = {
#     'default': {
#         'hosts': [
#             'PROJECTVM.tacc.utexas.edu',
#             'PROJECTVM.tacc.utexas.edu',
#         ],
#     },
#     'staging': { #dev/qa
#         'hosts':  [
#             'PROJECTVMSTAGING.tacc.utexas.edu',
#         ]
#     },
#     'dev': {
#         'hosts': [
#             'elasticsearch'
#         ]
#     },
#     'localhost': {
#         'hosts': [
#             'localhost'
#         ]
#     }
# }
```

</details>

Reply:
Removed comment from stache